src/pages/groupComparison/GroupComparisonMutationMapper.tsx

```diff
@@ -46,8 +76,119 @@ export default class GroupComparisonMutationMapper extends MutationMapper<
         return null;
     }
 
+    protected formatPaginationStatusText = (text: string) => {
```

Comment:
see if we can override this in the underlying library (rather than this kinda hacky editing of the initial output)